Nos/Views/NoteButton.swift

```diff
@@ -15,6 +15,12 @@ struct NoteButton: View {
     var hideOutOfNetwork: Bool
     var repliesDisplayType: RepliesDisplayType
 
+    /// Indicates whether the number of likes is displayed.
+    var showLikeCount: Bool
+
+    /// Indicates whether the number of reposts is displayed.
+    var showRepostCount: Bool
```

Comment:
I'd like these names to be consistent with the similar property in LikeButton. So either these would be `showsLikeCount` and `showsRepostCount`, or the LikeButton property would be `showCount`. I think I like `shows` more than `show` but we do have `displayRootMessage` below, so maybe we should just go with `show` and change the property on LikeButton.

...or maybe they should all start with `display`?